osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHitStrong.cs

```diff
@@ -53,6 +55,15 @@ public override bool OnReleased(TaikoAction action)
             return base.OnReleased(action);
         }
 
+        protected override void LoadComplete()
+        {
+            base.LoadComplete();
+
+            if (Samples.Count > 1)
+                // Removes the "normal" hitsound, leaving only the hitfinish one
+                Samples.RemoveAt(0);

```

Comment:
A hitobject must always play its normal sample. Stable adds in any additions depending on which key is expected.